api-scripts/patron/placing-holds/README.md

```diff
@@ -0,0 +1,99 @@
+## API test:
+EDS programmatically calls mod-patron using the API - 
+`POST /patron/account/{id}/instance/{instanceId}/hold`
+
+An example of instance-level request to mod-patron:
+
```

Comment:
API endpoint to which this request is made is missing?

Reply:
The API endpoint to which this request is made is above this line. I doubled the API endpoint to make it more readable